### ISSUE TITLE
[a11y]: Latest aChecker ruleset causing CI failures in v10

### ISSUE THREAD

**Issue by tw15egan** [carbon-design-system/carbon#16274]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

Latest

### Automated testing tool and ruleset

aChecker latest deployment

### Description

While @francinelucca was working on https://github.com/carbon-design-system/carbon/pull/16245, the latest ruleset pulled in from aChecker was causing CI failures. As a workaround, the ruleset was set to the March release, rather than the latest.

We need to update the aChecker rule back to `latest` and fix the a11y violations.

### Reproduction/example

CI

### Steps to reproduce

CI fails when running the latest ruleset from aChecker

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
